### PR TITLE
Added findWindowByClass

### PR DESCRIPTION
**Pull request by WebFreak001**
Also adds actualTitle to SimpleWindow which returns the title of the window as reported by the OS.

Did not test the windows implementation, but I wrote a bit of code like this today and it should work if my memory isn't totally terrible.

Example:
```d
import std.stdio;
auto window = findWindowByClass("x-terminal-emulator");
writeln("Terminal: ", new SimpleWindow(window).actualTitle);
```

### CODE REVIEW

File: simpledisplay.d [hunk 1]
```diff
@@ -1269,9 +1269,10 @@ class SimpleWindow : CapableOfHandlingNativeEvent, CapableOfBeingDrawnUpon {
 	this(NativeWindowHandle nativeWindow) {
 		version(Windows)
 			impl.hwnd = nativeWindow;
-		else version(X11)
+		else version(X11) {
 			impl.window = nativeWindow;
-		else version(OSXCocoa)
+			display = XDisplayConnection.get();
```
Comment (adamdruppe) [inline simpledisplay.d]: Why is this there?

Comment (WebFreak001) [reply simpledisplay.d]: actualTitle breaks otherwise if you don't have any SimpleDisplay windows and you try to get the title of an unknown window

Comment (adamdruppe) [reply simpledisplay.d]: it should prolly just be a local variable since display can change in the event loop... maybe my mistake tho.

**Comment by WebFreak001** [comment]
separating in two PRs as discussed in IRC

**Pull request closed without merging**